.travis.yml

```diff
@@ -73,6 +73,11 @@ script:
   echo "Running the integration tests";
   vendor/bin/phpunit tests/Integration/;
 
+- >
+  echo;
+  echo "Running the system tests";
+  vendor/bin/phpunit tests/System/;
```

Comment:
@xh3n1 Thanks for the change.

I'd like to stick to the "one change/topic per PR" policy. Could you please move this to a separate PR? This will keep the history (and git blame) clean and allow for clean reverts if needed. Thanks!

Reply:
@oliverklee Thanks, I added it as a separate PR.